README.md

```diff
@@ -204,7 +205,8 @@ atexit.register(sfx.stop)
 
 ```
 ERROR:root:Posting to SignalFx failed.
-SSLError: hostname 'api.signalfx.com' doesn't match either of '*.signalfuse.com', 'signalfuse.com'
+SSLError: hostname 'ingest.signalfx.com' doesn't match either of
```

Comment:
@udaysagar2177 Is this still valid. Given we moved from api to ingest, this limitation might not be there anymore. Can you check and update this please.